attributes/default.rb

```diff
@@ -31,7 +31,7 @@
 default["kafka"]["logrotate"]["frequency"] = 'daily'
 default["kafka"]["logrotate"]["rotate"] = 5
 
-default["kafka"]["server.properties"]["log.dirs"] = "/tmp/kafka-logs"
+default["kafka"]["server.properties"] = {}
```

Comment:
+1 any attributes that had meaning but are now meaningless or need to be set by consumers we should call out as part of an upgrade guide.